etcdserver/server.go

```diff
@@ -1187,6 +1201,15 @@ func (s *EtcdServer) swapKV(kv dstorage.ConsistentWatchableKV) dstorage.Consiste
 	return old
 }
 
+func (s *EtcdServer) RegisterPathHookFunc(path string, f PathHookFunc, data interface{}) {
```

Comment:
Then how should we add a path specific action? I thought adding a branch for "/2/enabled" is too adhoc.